vipps-order-management-api.md

```diff
@@ -18,80 +18,78 @@ END_METADATA -->
 
-Vipps Order Management enables you to communicate with your customers through
-the payment receipts in the Vipps app. The purpose of doing this is to give
+Order Management enables you to communicate with your customers through
+the payment receipts in the Vipps or MobilePay app. The purpose of doing this is to give
 your customers more convenience, better overview and a more compelling shopping
-experience when they use Vipps to pay for your products and services.
-Vipps Order management also enables you to draw customers back to your website
-or app from links on the Vipps receipt view.
+experience when they use Vipps MobilePay to pay for your products and services.
+Order management also enables you to draw customers back to your website
+or app from links on the receipt view.
 
 This functionality is available for
 [recurring](https://developer.vippsmobilepay.com/docs/APIs/recurring-api)
 and
-[direct payments](https://developer.vippsmobilepay.com/docs/APIs/ecom-api),
+[direct payments](https://developer.vippsmobilepay.com/docs/APIs/epayment-api),
 but not for
 [pass-through payments](https://developer.vippsmobilepay.com/docs/APIs/psp-api).
 
 API version: 2.3.0.
 
 ## Before you begin
 
-This document assumes you have signed up as an organization with Vipps and have
-retrieved your API credentials for
-[the Vipps test environment](https://developer.vippsmobilepay.com/docs/vipps-developers/test-environment)
+Sign up your organization and retrieve your API credentials for the
+[test environment](https://developer.vippsmobilepay.com/docs/vipps-developers/test-environment)
 from
 [portal.vipps.no](https://portal.vipps.no).
 
 ### Authentication
 
-All Vipps API calls are authenticated with an access token and an API subscription key.
+All API calls are authenticated with an access token and an API subscription key.
 See
 [Get an access token](https://developer.vippsmobilepay.com/docs/APIs/access-token-api#get-an-access-token), for details.
 
 ### HTTP headers
 
-We recommend using the standard HTTP headers for all requests.
+We recommend using the standard HTTP headers for all requests
 
 See [HTTP headers](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/http-headers)
 in the Getting started guide, for details.
 
-### OrderId and PaymentType
+### Reference and PaymentType
 
-The idea with order management is to add a Receipt or Category to a Vipps transaction made with the eCom or Recurring API. So, the receipt needs to be connected to a `OrderId`. The `OrderId` is what **you** use when either initiating an eCom payment or creating a recurring charge.
+Use order management to add a *Receipt* or *Category* to a transaction made with the ePayment, eCom, or Recurring API. The receipt is connected to a `OrderId` (called `reference` in the new ePayment API). The `OrderId` is what **you** use when either initiating an eCom payment or creating a recurring charge.
```

Comment:
```suggestion
Use order management to add a *Receipt* or *Category* to a transaction made with the ePayment, eCom, or Recurring API. The receipt is connected to a `orderId` (called `reference` in the new ePayment API). The `orderId` is what **you** use when either initiating an eCom payment or creating a recurring charge.
```

Reply:
I don't know if we can make this change because they have used `orderId` everywhere, also in the yaml file. I think it will be more confusing to the user to use `reference` in that case.

Reply:
That needs to be done in a different PR.